### ISSUE TITLE
[Bug]: Scrolling not work with Smithing Table or Brewing Stand

### ISSUE THREAD

**Issue by BioMasterZap** [mrshmllow/BetterRecipeBook#124]
### What happened?

I have scrolling enabled and I can use the scroll wheel to changes the pages of the recipe book for the Crafting Table and Furnace, but scrolling is not changing the pages for the Smithing Table or Brewing Stand.

Also, when I enable scroll in circles, I can press the arrow to go from first to last page and last to first page, but I cannot scroll use the scroll wheel to loop around.

### Modloader

Fabric (Default)

### Minecraft Version

1.20.4

### Mod Version

1.10.0

### Relevant log output

```shell
None
```

**Comment by mrshmllow** [comment]
Cannot replicate

**Comment by BioMasterZap** [comment]
Apologies. While working on a different issue, I realized that I reported this without testing in a separate profile like I normally do and that other mods were causing the issue.

With just Better Recipe Book, the scroll in circles and Smithing Table and Brewing Stand scrolling work fine. The Smithing Table and Brewing Stand scrolling issue only when REI is active; if REI is toggled off, both scrolling and scroll in circles work fine for the Smithing Table and Brewing Stand. Chances are most users won't have both Better Recipe Book and REI anyway. 

I still have not worked out what is causing my issue with the scroll in circles for the other interfaces (it works on the Smithing Table and Brewing Stand with REI off, but still won't work for Crafting Table and Furnace), but it does work with just Better Recipe Book and REI so some other mod is messing with it. Hopefully, this might at least help if someone else runs into scrolling issues with REI.
